projects/kit/src/lib/masks/time/maskito-time-params.ts

```diff
@@ -0,0 +1,7 @@
+import type {MaskitoTimeMode, MaskitoTimeSegments} from '@maskito/kit';
+
+export interface MaskitoTimeParams {
+    readonly mode: MaskitoTimeMode;
+    readonly timeSegmentMaxValues?: Partial<MaskitoTimeSegments<number>>;
+    readonly step?: number;
```

Comment:
I did this because if a developer wants to use `maskitoTimeOptionsGenerator` and `maskitoStringifyTime`/`maskitoParseTime` then he will need to pass an object with settings to several functions. Where will he get the interface/type in this case? It will be clearly easier to write `const config: MaskitoTimeParams = …` than `const config: Parameters<typeof maskitoTimeOptionsGenerator>[0] =`.

Reply:
👌